src/callbacks_step/summary.jl

```diff
@@ -20,9 +20,14 @@ function SummaryCallback(reset_threads = true)
         initialize_summary_callback(cb, u, t, integrator;
```

Comment:
Could you please adapt the docstring above?

Reply:
Something like [ce06f7d](https://github.com/trixi-framework/Trixi.jl/pull/2275/commits/ce06f7d568d8cb45dc1495587803bc7a4cb77b43)?

Reply:
Yes, thanks!